src/styles/field.scss

```diff
@@ -0,0 +1,47 @@
+.field {
+    width: 610px;
+    display: flex;
+    justify-content: start;
+    align-items: center;
+    box-sizing: border-box;
+    border-radius: 2px;
+    background-color: $field-color;
+    &__text {
+        width: 100%;
+        box-sizing: border-box;
```

Comment:
* {
box-sizing: border:vox
}

Reply:
added reseat.scss

Reply:
reset.css

Reply:
added correctly, but here I made mistake in name